src/components/URLInput/URLInput.js

```diff
@@ -58,8 +61,10 @@ function URLInput(props) {
           return Promise.reject(error);
         }
         props.setRequest(data);
+        props.setLoading(false);
```

Comment:
All looks great.
I can try and help on the failing checks if you like?

Reply:
Thanks! I spotted it just as I uploaded it - have committed the fix (removed the unused react components). Should be good to go if you're happy? :)

Reply:
Definitely, off we go!